plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -742,10 +742,12 @@ private void addSplitPluginDependencies(String thisPlugin, MavenRunner.Config mc
             Map<String,VersionNumber> toReplace = new HashMap<String,VersionNumber>();
             Map<String,VersionNumber> toAddTest = new HashMap<String,VersionNumber>();
             Map<String,VersionNumber> toReplaceTest = new HashMap<String,VersionNumber>();
-            overridePlugins.forEach(plugin -> {
-                toReplace.put(plugin.getName(), plugin.getVersion());
-                toReplaceTest.put(plugin.getName(), plugin.getVersion());
-            });
+            if (overridePlugins != null) {
```

Comment:
And you'd get rid of this then if overridePlugins can NOT be null, which I think should really be always aimed at in 99% of the cases.